samba/translations/en.yaml

```
@@ -27,6 +27,11 @@ configuration:
     name: Enable Compatibility Mode
     description: >-
       Enable this to use old legacy Samba protocols on the Samba add-on.
+  apple_compatibility_mode:
+    name: Enable Compatibility Settings for Apple Devices
+    description: >-
+      Enable Samba configurations to improve interoperability with Apple devices.
+      May cause issues with file systems that do not support xattr such as exFAT.
```

This needs line breaks to stay within the 80 character line length limit of YAMLLint, so:

```suggestion
      Enable Samba configurations to improve interoperability with Apple
      devices. May cause issues with file systems that do not support xattr
      such as exFAT.
```